### ISSUE TITLE
Can‘t find ChromatiCraft glowingcliffs biome at ATG world

### ISSUE THREAD

**Issue by KuroPeach** [ReikaKalseki/Reika_Mods_Issues#2768]
It may be necessary to change the type of cliffs to the type of the formation of ATG
https://github.com/ReikaKalseki/ChromatiCraft/blob/ff3160e3902d5aeb88116d1210a2edf8fafc3d33/ChromatiCraft.java#L630
20000x20000，can‘t find blue color .
![image](https://user-images.githubusercontent.com/19620848/111332807-11247c00-86ad-11eb-95f0-95135583de91.png)
![image](https://user-images.githubusercontent.com/19620848/111333246-7a0bf400-86ad-11eb-9987-4e34ee682af9.png)

**Comment by KuroPeach** [comment]
Even if I set the probability to 1.0 in the ATG configuration file, there is still no
![image](https://user-images.githubusercontent.com/19620848/111333668-db33c780-86ad-11eb-91fd-0c8a2c77ad78.png)

        ATGBiomes.addBiome(sea, "Ocean", BiomeGenBase.ocean, 1.0);
        ATGBiomes.addBiome(sea, "Deep Ocean", BiomeGenBase.deepOcean, 1.0);
        ATGBiomes.addBiome(land, "Plains", BiomeGenBase.plains, 1.0);
        ATGBiomes.addBiome(land, "Desert", BiomeGenBase.desert, 1.0);
        ATGBiomes.addBiome(land, "Forest", BiomeGenBase.forest, 1.0);
        ATGBiomes.addBiome(land, "Taiga", BiomeGenBase.coldTaiga, 1.0);
        ATGBiomes.addBiome(land, "Ice Plains", BiomeGenBase.icePlains, 1.0);
        ATGBiomes.addBiome(land, "Jungle", BiomeGenBase.jungle, 1.0);
        ATGBiomes.addBiome(land, "Shrubland", (BiomeGenBase)ATGBiomeList.shrubland.get(), 1.0);
        ATGBiomes.addBiome(land, "Boreal Forest", BiomeGenBase.taiga, 1.0);
        ATGBiomes.addBiome(land, "Tundra", (BiomeGenBase)ATGBiomeList.tundra.get(), 1.0);
        ATGBiomes.addBiome(land, "Steppe", (BiomeGenBase)ATGBiomeList.steppe.get(), 1.0);
        ATGBiomes.addBiome(land, "Savanna", BiomeGenBase.savanna, 1.0);
        ATGBiomes.addBiome(land, "Tropical Shrubland", (BiomeGenBase)ATGBiomeList.tropicalShrubland.get(), 1.0);
        ATGBiomes.addBiome(land, "Woodland", (BiomeGenBase)ATGBiomeList.woodland.get(), 1.0);
        ATGBiomes.addBiome(land, "Mesa", BiomeGenBase.mesa, 1.0);
        ATGBiomes.addBiome(coast, "Beach", BiomeGenBase.beach, 1.0);
        ATGBiomes.addBiome(coast, "Gravel Beach", (BiomeGenBase)ATGBiomeList.gravelBeach.get(), 1.0);
        ATGBiomes.addBiome(coast, "Snowy Beach", (BiomeGenBase)ATGBiomeList.snowyBeach.get(), 1.0);
        ATGBiomes.addBiome(coast, "Swampland", BiomeGenBase.swampland, 1.0);

**Comment by ReikaKalseki** [comment]
They are all "land" type, and the group seems a largely freeform string; I see no reason why "cliffs" is invalid.

**Comment by voidsong-dragonfly** [comment]
Because IIRC the cliffs category never spawns. I had good results with making them parts of oceans, but that set of configs was complicated.

**Comment by ReikaKalseki** [comment]
In that case I need more info.

**Comment by voidsong-dragonfly** [comment]
It required a heavily-nonstandard ATG config with biomes many times the size of what they were by default as well as putting luminous cliffs in a very particular range of ocean that made them like an archipelago, none of which I remember too clearly at this point as the configs were two years old or so.
It also required ocean regions to be substantially larger and cover more realistic portions of the map - 50% or so instead of a bunch of inland seas.

**Comment by ReikaKalseki** [comment]
No, I meant about mechanically how this config (and its API-side interface) works.

**Comment by KuroPeach** [comment]
Hi, Reika, do you want to try changing cliffs to beach. Registered as a beach, the terrain may be a little strange

			ATGBiomes.addBiome(ATGBiomes.BiomeType.coast, "Beach", glowingcliffs, 0.125);

**Comment by ReikaKalseki** [comment]
Define "a little strange".

**Comment by voidsong-dragonfly** [comment]
To put it bluntly they'd look like giant strips of cliffs, because IIRC that's the beach category.

**Comment by ReikaKalseki** [comment]
That is definitely not desirable.
